Microsoft.Azure.Cosmos/src/Telemetry/OperationInfo.cs

```diff
@@ -62,7 +62,7 @@ internal OperationInfo(string metricsName, string unitName)
             {
                 this.GreaterThan1Kb = responseSizeInBytes > ClientTelemetryOptions.OneKbToBytes;
             }
-            this.Consistency = consistency?.ToString();
+            this.Consistency = GetConsistencyString(consistency);
```

Comment:
nit: static methods should have class name.
```suggestion
            this.Consistency = OperationInfo.GetConsistencyString(consistency);
```